ls1intum/Artemis
# `Documentation`: Clear Sphinx build warnings

### Checklist
#### General



### Motivation and Context


`docs/dev/setup.rst` uses includes which are stored in `docs/dev/setup/*.rst`. Those were once not triggered by builds by renaming them from `.rst` to `.rst.txt` which apparently didn't work great as they got renamed in [this commit](https://github.com/ls1intum/Artemis/commit/97d57e218fa42b32751c8c3c01c4c232123fa8dcz).

### Description

So now I just added these files to the sphinx excluded files to clear the warnings in the sphinx builds.

### Steps for Testing

- Build the docs locally and look if there are no warning, but the page generated by `setup.rst` works.
- Alternatively, check this PR's read-the-docs preview:
https://artemis-platform--6155.org.readthedocs.build/en/6155/dev/setup/

### Review Progress






#### Code Review
#### Manual Tests
